tests/acceptance/relationships/belongs-to-test.js

```diff
@@ -1,4 +1,4 @@
-import { module, skip as test } from 'qunit';
```

Comment:
Waaat, this whole file was skipped?!

Reply:
yes, so was another one. Now only a few tests are skipped :)